src/PlaylistDatabase.cxx

```diff
@@ -25,7 +25,8 @@
 #include "util/StringStrip.hxx"
 #include "util/RuntimeError.hxx"
 
-#include <string.h>
```

Comment:
With `<string.h>` removed, who provides `strcmp()`?

Reply:
Page 705 of the standard says:

The contents and meaning of the header<cstring>are the same as the C standard library header<string.h>

Reply:
Aye. Then you still need to convert some `<string.h> to `<cstring>`, because only the latter provides the `std::` namespace.

Reply:
Some other header provides it.

Reply:
Which other header? Don't rely on indirect includes, unless they are guaranteed.